wsd/wopi/WopiStorage.cpp

```diff
@@ -867,7 +867,7 @@ WopiStorage::handleUploadToStorageResponse(const WopiUploadDetails& details,
             }
             else
             {
-                LOG_ERR("Invalid or missing JSON in " << wopiLog << " HTTP_OK response.");
+                LOG_ERR("Invalid or missing JSON in " << wopiLog << " HTTP_OK response. Expected json object with a LastModifiedTime value");
```

Comment:
The error happens because there is no text in the answer and we expect json, hence the parse failure.

This will point the host implementer in the right direction.

I would argue this should be a Warning in the first place

Reply:
yes it should be a warning.

Reply:
Just changed it.

The drawback is the implementor might just take seriously enough the warn.